pkg/account/component.go

```diff
@@ -15,6 +16,12 @@ import (
 	kc "github.com/cloudtrust/keycloak-client"
 )
 
+// Constants
+const (
+	ActionVerifyEmail       = "VERIFY_EMAIL"
```

Comment:
why 2 diff styles?

Reply:
Yes... one comes from Keycloak, the other one comes from 2019.kc-sms